api/capi/include/platform/ml-api-common.h

```diff
@@ -0,0 +1,64 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * NNStreamer API / Tizen Machine-Learning API Common Header
+ * Copyright (C) 2020 MyungJoo Ham <myungjoo.ham@samsung.com>
+ */
+/**
+ * @file	ml-api-common.h
+ * @date	07 May 2020
+ * @brief	Dummy ML-API Common Header from nnstreamer
+ * @see		https://github.com/nnstreamer/nnstreamer
+ * @author	MyungJoo Ham <myungjoo.ham@samsung.com>
+ * @author Jijoong Moon <jijoong.moon@samsung.com>
+ * @bug		No known bugs except for NYI items
+ *
+ * @details
+ *      More entries might be migrated from nnstreamer.h if
+ *    other modules of Tizen-ML APIs use common data structures.
+ */
+#ifndef __ML_API_COMMON_H__
+#define __ML_API_COMMON_H__
+
+#ifdef __TIZEN__
+
+#error TIZEN build should not use this file.
+
+#else
```

Comment:
#endif. Try to lower the if/then/else depth.

Reply:
Much better. Thank you!